examples/rbac-proxy-resources.yaml

```diff
@@ -0,0 +1,184 @@
+- kind: ServiceAccount
+  name: piraeus-op
+  namespace: linstor
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: linstor-controller-rbac-proxy
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: piraeus-rbac-proxy
+subjects:
+- kind: ServiceAccount
+  name: linstor-controller
+  namespace: linstor
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: linstor-satellite-rbac-proxy
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: piraeus-rbac-proxy
+subjects:
+- kind: ServiceAccount
+  name: linstor-satellite
+  namespace: linstor
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: piraeus-operator
+  namespace: d8-monitoring
+  labels:
+    prometheus: main
+spec:
+  podMetricsEndpoints:
+  - targetPort: 8080
+    scheme: https
+    bearerTokenSecret:
+      name: "prometheus-token"
+      key: "token"
+    tlsConfig:
+      insecureSkipVerify: true
+  selector:
+    matchLabels:
+      name: piraeus-op-operator
+  namespaceSelector:
+    matchNames:
+    - linstor
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: linstor-controller
+  namespace: d8-monitoring
+  labels:
+    prometheus: main
+spec:
+  podMetricsEndpoints:
+  - targetPort: 3370
+    scheme: https
+    bearerTokenSecret:
+      name: "prometheus-token"
+      key: "token"
+    tlsConfig:
+      insecureSkipVerify: true
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: piraeus-op-cs
+      app.kubernetes.io/managed-by: piraeus-operator
+      app.kubernetes.io/name: piraeus-controller
+  namespaceSelector:
+    matchNames:
+    - linstor
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: linstor-node
+  namespace: d8-monitoring
+  labels:
+    prometheus: main
+spec:
+  podMetricsEndpoints:
+  - targetPort: 9942
+    scheme: https
+    bearerTokenSecret:
+      name: "prometheus-token"
+      key: "token"
+    tlsConfig:
+      insecureSkipVerify: true
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: piraeus-op-ns
+      app.kubernetes.io/managed-by: piraeus-operator
+      app.kubernetes.io/name: piraeus-node
+  namespaceSelector:
+    matchNames:
+    - linstor
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: access-to-linstor
+  namespace: linstor
+rules:
+- apiGroups: ["apps"]
+  resources: ["daemonsets/prometheus-metrics", "deployments/prometheus-metrics"]
+  resourceNames: ["linstor-controller", "linstor-node", "piraeus-operator"]
+  verbs: ["get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: access-to-linstor
+  namespace: linstor
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: access-to-linstor
+subjects:
+- kind: User
+  name: d8-monitoring:scraper
+- kind: ServiceAccount
+  name: prometheus
+  namespace: d8-monitoring
```

Comment:
Basically same as above here. kube-prometheus "normal" destination is the `monitoring` namespace so we should use that here and mention it in a comment at the start.

Reply:
fixed, thanks